config/localTemplate.go

```diff
@@ -471,8 +471,8 @@ type Local struct {
 	// NetworkMessageTraceServer is a host:port address to report graph propagation trace info to.
 	NetworkMessageTraceServer string `version[13]:""`
 
-	// VerifiedTranscationsCacheSize defines the number of transactions that the verified transactions cache would hold before cycling the cache storage in a round-robin fashion.
-	VerifiedTranscationsCacheSize int `version[14]:"30000" version[23]:"150000"`
+	// VerifiedTransactionsCacheSize defines the number of transactions that the verified transactions cache would hold before cycling the cache storage in a round-robin fashion.
+	VerifiedTransactionsCacheSize int `version[14]:"30000" version[23]:"150000"`
```

Comment:
I imagine changing this would require changing it _everywhere_.
https://github.com/search?q=repo%3Aalgorand%2Fgo-algorand%20VerifiedTranscationsCacheSize&type=code